vendor/modules.txt

```diff
@@ -117,14 +117,14 @@ gopkg.in/inf.v0
 gopkg.in/yaml.v2
 # k8s.io/api v0.0.0-20190313235455-40a48860b5ab
 k8s.io/api/apps/v1
-k8s.io/api/apps/v1beta1
 k8s.io/api/autoscaling/v2beta1
 k8s.io/api/batch/v1
 k8s.io/api/batch/v1beta1
 k8s.io/api/certificates/v1beta1
 k8s.io/api/core/v1
 k8s.io/api/extensions/v1beta1
 k8s.io/api/policy/v1beta1
+k8s.io/api/apps/v1beta1
```

Comment:
Just curious why go mods do this change, moves the import in this file. The pkg is not used anymore in the code directly, guessing its used by one of the other deps e.g. client-go internally, but it makes me wonder if we can bump the version of client-go and in turn get rid of these dependencies. (But that might be going down the rabbit hole of solving deps, and probably falls outside of this PR :)

Reply:
That makes sense. But yes, this definitely is out of scope for this PR. I believe we have k8s 1.14 depped into kube state metrics. We are just behind the latest by a patch release.